test/plugins/base.plugin.bats

```diff
@@ -2,7 +2,10 @@
 
 load ../test_helper
 load "${BASH_IT}/vendor/github.com/erichs/composure/composure.sh"
+load ../../lib/log
+load ../../lib/helpers
 load ../../plugins/available/base.plugin
+export BASH_IT_LOAD_PRIORITY_SEPARATOR='---'
```

Comment:
why do you need to export it here? isn't it defined in the helper file?

Reply:
It's not loaded, it seems. The tests don't load the whole kit, just bits and pieces, and I was getting errors on this one. I exported because I'm unsure about BATS' use of subshells.